[2023-03-21T00:56:16.736+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: project_masterdag.delete_dataproc_cluster manual__2023-03-21T00:43:30.185105+00:00 [queued]>
[2023-03-21T00:56:16.778+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: project_masterdag.delete_dataproc_cluster manual__2023-03-21T00:43:30.185105+00:00 [queued]>
[2023-03-21T00:56:16.779+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T00:56:16.780+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-03-21T00:56:16.780+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T00:56:16.831+0000] {taskinstance.py:1303} INFO - Executing <Task(DataprocDeleteClusterOperator): delete_dataproc_cluster> on 2023-03-21 00:43:30.185105+00:00
[2023-03-21T00:56:16.851+0000] {standard_task_runner.py:55} INFO - Started process 6333 to run task
[2023-03-21T00:56:16.871+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project_masterdag', 'delete_dataproc_cluster', 'manual__2023-03-21T00:43:30.185105+00:00', '--job-id', '281', '--raw', '--subdir', 'DAGS_FOLDER/project_master_dags.py', '--cfg-path', '/tmp/tmp9u1f4iwd']
[2023-03-21T00:56:16.882+0000] {standard_task_runner.py:83} INFO - Job 281: Subtask delete_dataproc_cluster
[2023-03-21T00:56:16.928+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:250: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-03-21T00:56:17.135+0000] {task_command.py:388} INFO - Running <TaskInstance: project_masterdag.delete_dataproc_cluster manual__2023-03-21T00:43:30.185105+00:00 [running]> on host 796cac132c25
[2023-03-21T00:56:17.494+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_masterdag
AIRFLOW_CTX_TASK_ID=delete_dataproc_cluster
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T00:43:30.185105+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T00:43:30.185105+00:00
[2023-03-21T00:56:17.498+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-03-21T00:56:17.499+0000] {dataproc.py:901} INFO - Deleting cluster: my-dataproc-cluster
[2023-03-21T00:57:29.921+0000] {dataproc.py:871} INFO - Cluster deleted.
[2023-03-21T00:57:29.941+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=project_masterdag, task_id=delete_dataproc_cluster, execution_date=20230321T004330, start_date=20230321T005616, end_date=20230321T005729
[2023-03-21T00:57:29.990+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-21T00:57:30.015+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
